addons/mail/static/src/discuss/core/common/channel_invitation.js

```diff
@@ -200,4 +204,11 @@ export class ChannelInvitation extends Component {
         }
         return _t("Invite");
     }
+
+    get headerInfo() {
+        return {
+            title: this.props.showHeader ? _t("Notification Settings") : "",
+            icon: this.props.showHeader ? "fa fa-bell" : "",
+        };
+    }
```

Comment:
Looks wrong: this is channel invitation panel. This change show the wrong panel name:
![Screenshot 2025-05-23 at 15 33 35](https://github.com/user-attachments/assets/8e4be66f-43ca-4711-aeac-0e1c7f2a1154)

I also think that I would keep showing the panel header in all context. The dialog name should be the name of conversation, so that we know for sure this is the channel action dialog on the right thread. People can fat-finger / miss-click a sibling conversation and make a mistake!